app/src/cpp/cfc-cpp/jni.cpp

```diff
@@ -53,7 +53,7 @@ namespace {
 
 		cv::Scalar color = cv::Scalar(255,255,255);
 		if (in_progress == 1)
-			color = cv::Scalar(255,100,100);
+			color = cv::Scalar(255,244,94); // 0,191,255
```

Comment:
As you can see, blue is also an option. I decided to throw our colorblind friends a bone and not use red vs green.